src/ui/public/courier/courier.js

```diff
@@ -131,9 +131,9 @@ uiModules.get('kibana/courier')
     };
 
     // Listen for refreshInterval changes
-    $rootScope.$watchCollection('timefilter.refreshInterval', function () {
-      var refreshValue = _.get($rootScope, 'timefilter.refreshInterval.value');
-      var refreshPause = _.get($rootScope, 'timefilter.refreshInterval.pause');
+    $rootScope.$watchCollection('$$timefilter.refreshInterval', function () {
```

Comment:
Yeah, we can't use the `$$timefilter` here. The `$$` prefix is explicitly designed to prevent it's use. Maybe we can inject the timefilter and use a getter function in the `watchCollection`